web/src/client/storage.js

```diff
@@ -25,145 +25,687 @@ import DBusClient from "./dbus";
 import { WithStatus, WithValidation } from "./mixins";
 import cockpit from "../lib/cockpit";
 
-const STORAGE_SERVICE = "org.opensuse.DInstaller.Storage";
-const STORAGE_PATH = "/org/opensuse/DInstaller/Storage1";
-const PROPOSAL_CALCULATOR_IFACE = "org.opensuse.DInstaller.Storage1.Proposal.Calculator";
-const PROPOSAL_IFACE = "org.opensuse.DInstaller.Storage1.Proposal";
+/**
```

Comment:
Disclaimer: I know this file is growing too much. Probably some of this classes can be extracted to a separate file. For example, I expect `ObjectManagerIface` will be required by others clients, too. For now, I would keep everything in this file. We can extract classes later when it is really needed.